vscode-trace-extension/src/extension.ts

```diff
@@ -15,9 +15,11 @@ import { VSCODE_MESSAGES } from 'vscode-trace-common/lib/messages/vscode-message
 import { TraceViewerPanel } from './trace-viewer-panel/trace-viewer-webview-panel';
```

Comment:
This comment is about the commit message: the `Signed-off-by` is missing angle brackets around the email. You may check examples in the git log if needed.